app/javascript/packs/services_search_bar.js

```diff
@@ -0,0 +1,11 @@
+// TODO: remove this pack when apiap rolling updated is removed
```

Comment:
Add this somewhere in our pending jiras or create new one because otherwise it will be forgotten @macejmic